### DISCUSSION TITLE
Why are $ZSH_CUSTOM subdirectories not added to $fpath by default?

### DISCUSSION THREAD

**Discussion by wallacms** [ohmyzsh/ohmyzsh#12460] [Help] [answered]
I'd like to be able to keep all of my customizations to my zsh environment under $ZSH_CUSTOM.  There's already support for $ZSH_CUSTOM/plugins, is there a good reason not to automatically add $ZSH_CUSTOM/completions and $ZSH_CUSTOM/functions to the $fpath in oh-my-zsh.sh (as is done for $ZSH/functions and $ZSH/completions)?

Note that this question stems from the answer here: https://github.com/ohmyzsh/ohmyzsh/discussions/11342 - this works, but since completions is not in the .gitignore it causes git to see an uncommitted file and it means I can't use a separate git repo for my custom changes.

**Comment by MartinHammarstedt** [comment, marked as answer]
In case you missed it, this was fixed in commit 6754b7e67b71511c8c06b75be6d4d06a386acbda.
